collector/database/chart.php

```diff
@@ -113,8 +122,10 @@
     $freqoptions,
     $selectedfrequency
 );
+$checkboxes[] = ['freq' => html_writer::empty_tag('input',
```

Comment:
This appears a little dodgy. It is a single item tacked on to the end of an array it doesn't belong to. It should be added to the template context as its own entity.